circe/src/test/scala/io/github/mcgizzle/circe/CirceDecoderTests.scala

```diff
@@ -8,26 +8,25 @@ import shapeless.test.illTyped
 class CirceDecoderTests extends FlatSpec with Matchers {
 
   "Decoder" should "provide an implicit Decoder instance given a circe one" in {
-    import io.circe.generic.auto._
+      import io.circe.generic.auto._
```

Comment:
why all this whitespace lol

Reply:
trying out scalafmt, haven't got the config file setup just right yet it seems